slices/libjq1.yaml

```diff
@@ -0,0 +1,9 @@
+package: libjq1
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libonig5_libs
+    contents:
+      /usr/lib/*-linux-gnu/libjq.so.**:
```

Comment:
Should `gnu` be wildcarded here as well?

Reply:
Yeah you are right because of ``armhf``